CHANGES.rst

```diff
@@ -75,6 +75,10 @@ Changed
 - Make sure that the :command:`systemd` Tinc units correctly execute the
   :command:`tinc-down` scripts on service shutdown. [drybjed_]
 
+- Change the naming scheme of the debops.ferm_ configuration files in
+  anticipation of the upcoming changes. You might need to remove the old
+  firewall configuration files to avoid duplicate rule entries. [drybjed_]
```

Comment:
The `debops.tinc` role doesn't have any idea how the firewall is configured, `debops.ferm` does. I suppose that the `ferm` role could have some tasks that allow to move files around according to rules defined via dependent variables... But that becomes not needed at all on a fresh host, or after the upgrade is performed. The project is still in a beta phase, and hopefully the file structure used by the `debops.ferm` role won't change much after the incomigng rewrite, so is that really needed?